src/pg_to_evalscript/javascript_processes/array_find.js

```diff
@@ -0,0 +1,19 @@
+function array_find(arguments) {
+  const { data, value } = arguments;
+  if (data === null || data === undefined) {
```

Comment:
The new `validateParameter` function should be used here for checking arguments (and tests should then be changed appropriately)

Reply:
Looks good now, resolving